src/server/api/boards/boards.js

```diff
@@ -100,6 +100,47 @@ function addCardToColumn(board, columnId, cardName) {
   });
 }
 
+function validateColumnId(board, columnId, newColumnId) {
+  let oldFound = false;
+  let newFound = false;
+
+  board.columns.forEach(function(element) {
+    if (element._id.toString() === columnId.toString()) {
+      oldFound = true;
+    }
+    if (element._id.toString() === newColumnId.toString()) {
+      newFound = true;
+    }
+  });
+  return oldFound && newFound;
+}
+
+function moveCard(board, request, columnId, cardId) {
+  let card = {};
+
+  board.columns = board.columns.map(function(element) {
+    if (element._id.toString() === columnId.toString()) {
```

Comment:
do you really have to convert the id to string explicitly every time? doesn't it have like a `.equals` method or something?

Reply:
if it doesn't have one you can create your own method that returns a boolean value (`compareId(id1, id2)`)